packages/nimble/R/genCpp_operatorLists.R

```diff
@@ -103,7 +103,7 @@ brackOperators <- list('[' = c('[',']'), '[[' = c('[[',']]'))
 
 ## see distributions_processInputList for some relevant lists of distributions functions 
 
-callToSkipInEigenization <- c('copy','setValues', 'setValuesIndexRange', 'getValues', 'getValuesIndexRange', 'setSize', 'resize', 'getsize', 'size', 'resizeNoPtr','assert', 'return', 'blank', 'rankSample', 'nimArr_dmnorm_chol', 'nimArr_dmvt_chol', 'nimArr_dwish_chol', 'nimArr_dmulti', 'nimArr_dcat', 'nimArr_dinterval', 'nimArr_ddirch', 'nimArr_rmnorm_chol', 'nimArr_rmvt_chol', 'nimArr_rwish_chol', 'nimArr_rmulti', 'nimArr_rcat', 'nimArr_rinterval', 'nimArr_rdirch', 'calculate', 'calculateDiff', 'simulate', 'getLogProb', 'nimEquals', 'startNimbleTimer', 'endNimbleTimer')
```

Comment:
nit: This line would be easier to review if it were split into multiple lines, e.g.
```
callToSkipInEigenization <- c(
    'copy',
    'setValues',
    'setValuesIndexRange',
    'getValues',
    'getValuesIndexRange',
   ...)
```